packages/javascript/jupyterlab-plotly/src/Figure.ts

```diff
@@ -826,9 +826,13 @@ export class FigureView extends DOMWidgetView {
     this.model.on("change:_py2js_update", this.do_update, this);
     this.model.on("change:_py2js_animate", this.do_animate, this);
 
-    // MathJax configuration
+    // MathJax v2 configuration
     // ---------------------
-    if ((window as any).MathJax) {
+    if (
+      (window as any).MathJax &&
+      (window as any).MathJax.Hub &&
+      (window as any).MathJax.Hub.Config
```

Comment:
As this is a more modern toolchain, we should be able to use [optional chaining](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining):
```js
if ((window as any)?.MathJax?.Hub?.Config)
```

And in fact, if we don't branch off for v3 you can even use the function call version:
```js
(window as any)?.MathJax?.Hub?.Config?.({ SVG: { font: "STIX-Web" } });
```

Reply:
Done in 13f7c91f7ba0c89ded4383b35ac0ffc46f563060.